### PR TITLE
Update proxy

### PR DESCRIPTION
**Pull request by kyessenov**
Changes in mixer client and envoy config schema.

### CODE REVIEW

File: proxy/envoy/testdata/envoy-fault.json.golden [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "listeners": [
     {
-      "port": 80,
+      "address": "tcp://0.0.0.0:1081",
```
Comment (wattli) [inline proxy/envoy/testdata/envoy-fault.json.golden]: Why change port to 1081?

**Comment by kyessenov** [comment]
#382

File: proxy/envoy/testdata/envoy-fault.json.golden (hunk 1 above)
Comment (kyessenov) [inline proxy/envoy/testdata/envoy-fault.json.golden]: List is sorted by address and the order changed.

Comment (wattli) [inline proxy/envoy/testdata/envoy-fault.json.golden]: i c, thanks

**Pull request merged**